.github/workflows/testing.yml

```diff
@@ -8,20 +8,37 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
-        os: [ ubuntu-latest ]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
3.12?

Reply:
Will be removed as tests were failing - will create an issue to make it work :+1: